CHANGELOG.md

```diff
@@ -1,3 +1,19 @@
+## Version 0.2.6 (12-28-2022)
+
+### Security
+ - Introduce API wrapper for custom themes and modules to facilitate easier CSRF tokens (#612)
+
+## Bugfixes
+ - Fixed an issue with alias under Apache2 (#626)
+ - Add an additional check for CSFR when php://input is empty (#626)
+ - Fixed an issue where you where not able to reuse existing packages as root user (#607) 
+ - Refactor loading locales (#623)
+ - Update how we load available locale selection (#611)
+ 
+### Other
+ - Updated some dependencies.
+ - Sync up locales
```

Comment:
Let's specifically note that FOSSBilling is 21% translated into 13 different languages and provide a link to https://translate.fossbilling.org/ for anyone who wants to help translate